R/example.R

```diff
@@ -195,4 +195,57 @@ example_large_sized_commits <- function(folder_path="/tmp", folder_name) {
 }
 
 
+#' Example fake Mbox data with all parameters
+#'
+#' Creating a fake mbox with all parameters filled in, to ensure the function generates mbox properly
+#'
+#' @param folder_path Default folder path set to "/tmp"
+#' @param folder_name Name of folder where .mbox file will be stored
+#' @return Folder path of .mbox sample file that was created
+#' @export
+example_mbox_normal <- function(folder_path = "/tmp", folder_name) {
```

Comment:
We need an example that shows how to create at least two e-mails on the mailing list. Otherwise, we can only simulate mailing lists of 1 email!